sgxs-tools/src/sgx_detect/interpret.rs

```diff
@@ -354,3 +356,26 @@ mod serde {
         }
     }
 }
+
+#[derive(Debug, Clone, Copy, Serialize, Deserialize)]
+pub enum DSStatus {
```

Comment:
I don't think these and the types below belong in this module

Reply:
@jethrogb Aren't these similar to `KmodStatus`, `SgxEnableStatus` and `AesmStatus` which have also been defined in the same module ?

Reply:
Yeah you're right. Then maybe only `BuildType` shouldn't be here.